packages/core/register/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../../.eslintrc.json"
```

Comment:
There's `@parcel/eslint-config` package you can extend so you can avoid using relative paths.

Reply:
Perfect, I'll update it